### PR TITLE
Update to latest version that is only a simple go binary

### PR DESCRIPTION
**Pull request by emiljoha**

### CODE REVIEW

File: se.emijoh.mpw.yml [hunk 1]
```diff
@@ -1,40 +1,19 @@
...
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+command: mpw
 modules:
-  - name: mpwcli
+  - name: mpw
     buildsystem: simple
     build-commands:
-      - install -D mpwcli.py /app/bin/mpwcli.py
+      - cd go && /usr/lib/sdk/golang/bin/go build -v .
```
Comment (hfiguiere) [inline se.emijoh.mpw.yml:12]: will this work offline?

Comment (emiljoha) [reply se.emijoh.mpw.yml:12]: It should. I use the vendoring patterns so I have all the dependencies in the repo under a vendor directory. https://github.com/emiljoha/mpw/tree/master/go/vendor

Might of course messed it up somehow :shrug:

**Comment by emiljoha** [comment]
@hfiguiere The PR check status does not seem to update properly. Any idea what might be wrong?

**Comment by hfiguiere** [comment]
it's called patience. All of this run in the cloud on many computers. It's like traffic: a bottleneck can slow everything down, and using the horn doesn't help.

**Comment by emiljoha** [comment]
Alright, I'll try some of that patience thing, pretty sure it is not my cup of tea though. 

Because I got the build successful comment and the link in the comment showed a build that was done I expected the check to have a status reported.

https://github.com/flathub/se.emijoh.mpw/pull/6#issuecomment-1894420573
image

**Pull request merged**